src/components/Navigation.spec.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import Navigation from './Navigation.js';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Form', () => {
+  it('renders two navigation buttons', () => {
```

Comment:
It actually renders two links and not buttons.
```suggestion
  it('renders two navigation links', () => {
```